ddp-workspace/projects/ddp-testboston/src/assets/i18n/fr.json

```diff
@@ -500,7 +500,10 @@
     },
     "Title": {
       "Default": "TestBoston",
-      "Alert": "Votre session va bientôt expirer"
+      "Alert": "Votre session va bientôt expirer",
+      "Dashboard": "TestBoston Participant Dashboard",
+      "UPS": "Covid at-home test and shipping instructions for TestBoston",
+      "Home": "TestBoston - The at-home COVID-19 testing study"
```

Comment:
If we don't have a translation, let's figure out a way to track it . Maybe we do the same thing we do in back-end, addding the [FR] / [ES] prefix?

Reply:
Will ask Jen.